backend/models/address.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+// Define address model
+const addressSchema = new Schema({
+	street: {
+		type: String,
+		maxlength: 250,
+	},
+	city: {
+		type: String,
+		maxlength: 250,
+	},
+	zip: {
+		type: Number,
+		max: 20,
```

Comment:
Look in to `max` vs `maxlength` and `String` vs `Number` for `zip`

Reply:
Noted and fixed.